.github/workflows/qokit-python-only.yml

```diff
@@ -5,9 +5,10 @@ name: QOKit test (python simulator only)
 
 on: # workflow_dispatch
  push:
-   branches: [ "main", "**" ]
- pull_request:
-   branches: [ "main" ]
+   #branches: [ "main", "**" ]
```

Comment:
Is there a reason the lines are commented out and not simply deleted?

Reply:
I commented those lines to test the GitHub Action for publishing. I have
- reverted the commented lines for 
- .github/workflows/qokit-package.yml
- .github/workflows/qokit-python-only.yml
Deleted pypi-test.yml and merged that code in pypi-test-publish.yml
Modified pypi-publish.yml to publish to pypi on release tag

I have updated codes and all actions are green